### PR TITLE
[docs][cloud] Login methods

### PR DESCRIPTION
**Pull request by ddhodge**
Social login methods

@netlify /preview/yugabyte-cloud/cloud-admin/manage-access/

### CODE REVIEW

**Review by sandeep-yb — APPROVED** [review]
LGTM

File: docs/content/preview/yugabyte-cloud/cloud-admin/manage-access.md [hunk 1]
```diff
@@ -33,12 +33,24 @@ Admin users have full access to all features. There must always be at least one
...
+- delete or change the role of other users
+- change login methods
+- create a billing profile
+- view account activity
 
 To access a cluster database, you need to ask an Admin for the username and password of a [database user created on your behalf](../../cloud-secure-clusters/add-users/).
 
+## Manage login methods
+
+Users can log in to YugabyteDB managed using either an email-based account or a social login. The available social logins include Google, GitHub, and LinkedIn. All three are enabled by default. Only an Admin user can modify the login methods.
```
Comment (lizayugabyte) [inline docs/content/preview/yugabyte-cloud/cloud-admin/manage-access.md:46]: YugabyteDB managed  >  YugabyteDB Managed

**Review by lizayugabyte — APPROVED** [review]

File: docs/content/preview/yugabyte-cloud/cloud-admin/manage-access.md (hunk 1 above)
Comment (kc-yb) [reply docs/content/preview/yugabyte-cloud/cloud-admin/manage-access.md:46]: We need to mention that if the admin has revoked a particular login method (say Google), users can use the "Forgot password" option to configure email based login (if not configured already)

**Review by jagdish-yb — APPROVED** [review]
LGTM

**Pull request merged**